src/Core/Notifications/IDisabledNotificationsStorage.cs

```diff
@@ -105,7 +105,8 @@ private DisabledNotifications ReadDisabledNotifications()
             {
                 if (!fileSystem.File.Exists(FilePath)) 
                 { 
-                    logger.LogDebug($"[Notifications] Disabled notifications file does not exist. File: {FilePath}"); 
+                    logger.LogDebug($"[Notifications] Disabled notifications file does not exist. File: {FilePath}");
+                    fileSystem.Directory.CreateDirectory(Path.GetDirectoryName(FilePath));
```

Comment:
why not do it on save? this way if the user never dismisses anything, we won't create an empty directory.

Reply:
So that we will create this once per user.

Reply:
But save will be seldom anyway, so it makes sense.